src/base/bittorrent/session.cpp

```diff
@@ -4954,10 +4955,11 @@ void Session::handleSessionStatsAlert(const lt::session_stats_alert *p)
     m_status.diskWriteQueue = stats[m_metricIndices.peer.numPeersDownDisk];
     m_status.peersCount = stats[m_metricIndices.peer.numPeersConnected];
 
-    const int numBlocksRead = stats[m_metricIndices.disk.numBlocksRead];
-    const int numBlocksCacheHits = stats[m_metricIndices.disk.numBlocksCacheHits];
+    // XXX: all of these should be std::int64_t once libtorrent <= 1.1.x is no longer supported
```

Comment:
Why can't we do it now?

Reply:
@glassez because libtorrent `RC_1_1` and `RC_1_2` have different types for them. Tbh I think I'm just going to give up on this change and leave it for later, for when `RC_1_1` support is dropped proper.